external/Boost.cmake

```diff
@@ -35,12 +35,10 @@
 #
 #######################################
 
-if(EXISTS ${REPOSITORY_DIR}/build/ThirdParty/share/boost_1_69_0.tar.gz)
-    set(BOOST_URL ${REPOSITORY_DIR}/build/ThirdParty/share/boost_1_69_0.tar.gz)
-    set(BOOST_HASH "9a2c2819310839ea373f42d69e733c339b4e9a19deab6bfec448281554aa4dbb")
+if(EXISTS "${REPOSITORY_DIR}/build/ThirdParty/share/boost.tar.gz")
+    set(BOOST_URL "${REPOSITORY_DIR}/build/ThirdParty/share/boost.tar.gz")
 else()
     set(BOOST_URL "https://dl.bintray.com/boostorg/release/1.69.0/source/boost_1_69_0.tar.gz")
-    set(BOOST_HASH "9a2c2819310839ea373f42d69e733c339b4e9a19deab6bfec448281554aa4dbb")
```

Comment:
don't we want to keep the hash checking? Ie if download breaks.

Reply:
I wanted to allow the manually downloaded package to be a different version.  Perhaps as a manual override type of thing.  I don't have a way to include the hash with the manually downloaded files.  This could be done I guess but it might be more trouble than it is worth.  If this turns out to be a problem I can re-visit this issue.

Reply:
I have lost track of the post that someone made about the air-gap issue.  If anyone knows where that was I wanted to respond to it saying we fixed it.

Reply:
https://discourse.numenta.org/t/running-community-fork-on-windows/5633/8?u=dmac